code/parse/sexp/LuaSEXP.cpp

```diff
@@ -518,6 +525,33 @@ void LuaSEXP::parseTable() {
 			type = get_parameter_type("string");
 		}
 
+		if ((strcmp(type.first.c_str(), "enum")) == 0) {
+			type.second = increment_list_id();
+			required_string("+Enum Name:");
+
+			SCP_string enum_name;
+			stuff_string(enum_name, F_NAME);
+
+			enum_list thisList;
+			thisList.name = enum_name;
+
+			SCP_vector<SCP_string> list_items;
+			while (optional_string("+Enum:")) {
+				SCP_string item;
+				stuff_string(item, F_NAME);
+				list_items.push_back(item);
+			}
+
+			if ((int)list_items.size() > 0) {
+				thisList.list = list_items;
```

Comment:
Make it ``thisList.list = std::move(list_items);``
This helps the compiler to optimize, since you tell it "I don't need list_items anymore, you are allowed to destroy it in the process of moving it to thisList.list". This may enable the compiler to avoid copying the list, and for example in this case automatically reusing some space.